### DISCUSSION TITLE
How to perform controlled migrations AKA an equivalent to SQLite pragma user_version

### DISCUSSION THREAD

**Discussion by trenta3** [duckdb/duckdb#6783] [Q&A]
SQLite has a modifiable pragma called [user_version](https://www.sqlite.org/pragma.html#pragma_user_version) which allows storing the database schema version in it. Does DuckDB have an equivalent of it? Otherwise what is the suggested way to embed custom metadata inside of the database?

**Comment by anacrolix** [comment]
I'm stuck on this too.
